### DISCUSSION TITLE
Unable to Access GitHub Copilot through Student Developer Pack

### DISCUSSION THREAD

**Discussion by ritvikpen** [community/community#114634] [GitHub Education]
### Hi everyone,

Hi, I just got access to benefits from the GitHub Student Developer Pack, but it seems like I can't get access to GitHub Copilot. The instructions stated: You'll see "You're eligible to use copilot for free" and a button to claim the offer. However, when I navigate to the Copilot tab in my settings, I don't see this prompt or button. Anyone know how to resolve this?

**Comment by samus-aran** [comment]
Hi @ritvikpen 👋 
Are you still experiencing this?

**Reply by MatthewBlyth** [reply]
Hi @samus-aran, 
I'm in the same situation as ritvikpen and I'm still experiencing the issue.

**Comment by Jossyboydgenius** [comment]
Same here, it's still telling me to pay.

**Comment by puppetsheep** [comment]
Same here, I always automatically redirect to [prices](https://github.com/pricing)

**Comment by dlight** [comment]
Same here, [the copilot settings](https://github.com/settings/copilot) doesn't display "You're eligible to use copilot for free" and has no button to claim the offer, despite this procedure being mentioned [in the student pack](https://education.github.com/pack#github-copilot). Only a pricing page and an organization list appears in the settings.

However, Copilot appear as a claimed offer [in Github education dashboard](https://education.github.com/learner/learn). And if I go to the [benefits application](https://education.github.com/discount_requests/application) page, it says "It is not necessary for you to reverify at this time."

Despite all that, when enabling the [Copilot VSCode extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot), it says "Invalid copilot token: missing token: 403" and "No access to GitHub Copilot found. You are currently logged in as dlight." in the log and notifications.

**Reply by Dipeshkstha** [reply]
i can't switch back camera on my phone to scan documents, while applying for student pack. pls help

**Reply by dlight** [reply]
Maybe ask somebody else to take the photos for you?

**Comment by paniert** [comment]
same issue

**Comment by howa003** [comment]
I had the same issue. [This discussion](https://github.com/orgs/community/discussions/58099) helped me.

**Reply by dlight** [reply]
The second one (going into billing and attempting to start a free trial of copilot individual) worked for me, thanks
